src/inputs/input/input.jsx

```diff
@@ -189,6 +190,19 @@ const propTypes = {
     ]),
 };
 
+// See https://www.educba.com/javascript-keycodes/ for a list of commonly used key codes
+const KEY_CODE_CTRL_KEY = 17;
+const KEY_CODE_LETTER_A = 65;
+const KEY_CODE_LETTER_C = 67;
+const KEY_CODE_LETTER_V = 86;
+const KEY_CODE_LETTER_X = 88;
+const KEY_CODE_LETTER_Y = 89;
+const KEY_CODE_LETTER_Z = 90;
+const KEY_CODE_NORMAL_NUMBERS_0 = 48;
+const KEY_CODE_NORMAL_NUMBERS_9 = KEY_CODE_NORMAL_NUMBERS_0 + 9;
+const KEY_CODE_NUMBER_PAD_0 = 96;
+const KEY_CODE_NUMBER_PAD_9 = KEY_CODE_NUMBER_PAD_0 + 9;
```

Comment:
Why not in the global constants?

Reply:
Yeah, I can move 'em there.  I guess I'll have to follow the existing convention too with `KEY_CODE` at the end.  (I kind of like it better like this at the beginning, but no biggie.)